crates/circuits/src/parse.rs

```diff
@@ -219,6 +221,33 @@ mod tests {
         assert_eq!(ciphertext, expected);
     }
 
+    #[test]
+    #[cfg(feature = "aes")]
+    #[ignore = "expensive"]
+    fn test_parse_aes_key_schedule() {
+        let circ = Circuit::parse("circuits/bristol/aes_128_key_schedule.txt").unwrap();
+        let (key, expected_ks, _, _) = aes_vectors();
+
+        let ks = <[u8; 176]>::from_lsb0_iter(circ.evaluate(key.iter_lsb0()).unwrap());
+
+        assert_eq!(expected_ks, ks);
+    }
+
+    #[test]
+    #[cfg(feature = "aes")]
+    #[ignore = "expensive"]
+    fn test_parse_aes_post_key_schedule() {
+        let circ = Circuit::parse("circuits/bristol/aes_128_post_key_schedule.txt").unwrap();
+        let (_, ks, msg, expected_out) = aes_vectors();
+
+        let out = <[u8; 16]>::from_lsb0_iter(
+            circ.evaluate(ks.iter_lsb0().chain(msg.iter_lsb0()))
+                .unwrap(),
+        );
+
+        assert_eq!(expected_out, out);
+    }
```

Comment:
These don't need to be in this module. The existing tests are for testing the _parsing_ not the circuits themselves.